requirements/libraries.txt

```diff
@@ -21,6 +22,7 @@ django-storages==1.7.1
 django-treebeard==4.2.0
 django-watchman==0.15.0
 edgegrid-python==1.0.10
+elasticsearch7
```

Comment:
We should pin a specific version here.

Reply:
Done

Reply:
A specific version of elasticsearch7 is now pinned.